code/datums/outfits/ert/ert.dm

```diff
@@ -0,0 +1,40 @@
+/datum/outfit/admin/ert/nanotrasen
+	name = "Nanotrasen Responder"
+
+	uniform = /obj/item/clothing/under/ert
+	suit = null
+	suit_store = null
+	belt = /obj/item/weapon/storage/belt/military
+	shoes = /obj/item/clothing/shoes/jackboots
+	accessory = /obj/item/clothing/accessory/storage/black_vest
+	gloves = null
+	id = /obj/item/weapon/card/id/ert
+	backpack = /obj/item/weapon/rig/ert
+
+	l_ear = /obj/item/device/radio/headset/ert
+	l_hand = /obj/item/weapon/gun/projectile/automatic/rifle/z8
+
+	belt_contents = list(
+		/obj/item/weapon/handcuffs,
+		/obj/item/weapon/shield/riot/tact,
+		/obj/item/ammo_magazine/a556,
+		/obj/item/ammo_magazine/a556,
+		/obj/item/ammo_magazine/a556/ap
+	)
+	
+/datum/outfit/admin/ert/nanotrasen/specialist
+	belt = null
+	belt_contents = null
+
+/datum/outfit/admin/ert/nanotrasen/specialist/post_equip(mob/living/carbon/human/H)
+	switch(alert("Do you want to be an engineering or medical specialist?", "Response Team", "Engineering", "Medical")
```

Comment:
That is not a good way of doing that.
Have different ghostroles for the different specialists.

Reply:
Got it

Reply:
Solved